scripts/test

```diff
@@ -9,5 +9,5 @@ ${PREFIX}pytest -n 3 --cov-report term-missing --cov-report=xml:coverage.xml --c
 ${PREFIX}black commitizen tests --check
 ${PREFIX}isort --check-only commitizen tests
 ${PREFIX}flake8 commitizen/ tests/
-${PREFIX}mypy commitizen/ tests/
+${PREFIX}mypy --install-types --non-interactive commitizen/ tests/
```

Comment:
could you describe a bit more on why these arguments are added? Thanks!

Reply:
it is to avoid [such kind of pipeline errors](https://github.com/commitizen-tools/commitizen/runs/7410357144?check_suite_focus=true#step:6:86) because of chardet's missing stubs for mypy.

Reply:
if that's the case, should we add types-chardet directly to our dev dependency in `pyproject.toml`? Or what would be the main difference between adding `type-chardet` and using `--install-types`

Reply:
you're right, I did some test and locally mypy wanted that command instead of managing all through pyproject .

I'll do a commit to see if pipeline works having package into pyproject, if not I'll revert it.

Reply:
`types-chardet` into pyproject works

Reply:
nice! then, let's use `types-chardet`